src/PrestaShopBundle/Form/Admin/Improve/Design/Pages/CmsPageType.php

```diff
@@ -122,30 +124,52 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ],
             ])
             ->add('meta_title', TranslatableType::class, [
+                'type' => TextWithRecommendedLengthType::class,
                 'required' => false,
                 'options' => [
+                    'required' => false,
+                    'recommended_length' => SeoSettings::RECOMMENDED_TITLE_LENGTH,
+                    'attr' => [
+                        'maxlength' => SeoSettings::MAX_TITLE_LENGTH,
+                    ],
                     'constraints' => [
                         new TypedRegex([
                             'type' => 'generic_name',
                         ]),
                         new Length([
-                            'max' => 255,
+                            'max' => SeoSettings::MAX_TITLE_LENGTH,
                             'maxMessage' => $this->trans(
                                 'This field cannot be longer than %limit% characters',
                                 'Admin.Notifications.Error',
-                                ['%limit%' => 255]
+                                ['%limit%' => SeoSettings::MAX_TITLE_LENGTH]
                             ),
                         ]),
                     ],
                 ],
             ])
             ->add('meta_description', TranslatableType::class, [
                 'required' => false,
+                'type' => TextWithRecommendedLengthType::class,
                 'options' => [
+                    'required' => false,
+                    'recommended_length' => SeoSettings::RECOMMENDED_DESCRIPTION_LENGTH,
+                    'attr' => [
+                        'maxlength' => SeoSettings::MAX_DESCRIPTION_LENGTH,
+                    ],
                     'constraints' => [
                         new TypedRegex([
                             'type' => 'generic_name',
                         ]),
+                        new Length([
+                            'max' => SeoSettings::MAX_DESCRIPTION_LENGTH,
+                            'maxMessage' => $this->trans(
+                                'This field cannot be longer than %limit% characters',
+                                'Admin.Notifications.Error',
```

Comment:
The domain is the third parameter and not the second. Isn't it ?

Reply:
No, because of https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Form/Admin/Type/TranslatorAwareType.php#L62

Yes, it's weird to mix parameters order but it's from an old release :/

Reply:
No, there was a confusion in the making so this Symfony FormType extends `TranslatorAwareType` for which `$this->trans()` inverts the 3rd and the 2nd parameter.

Indeed that looks bad 😞